tools/scripts/gitrelease.pl

```diff
@@ -52,10 +53,28 @@
 
 ENDOUT
 
+sub get_news_post_release_msg {
+  my $settings = shift();
+  my $ver = $settings->{parsed_next_version}->{release_string};
+  return "OpenDDS $ver is currently in development, so this list might change.";
+}
+
+sub get_news_release_msg {
+  my $settings = shift();
+  return "OpenDDS $settings->{version} was released on $settings->{timestamp}.";
+}
+
 my $insert_news_template_after_line = 1;
 sub insert_news_template($$) {
-  my $version = shift;
-  my $timestamp = shift;
+  my $settings = shift();
+  my $post_release = shift();
+
+  print("pr: $post_release\n");
```

Comment:
Is this for debugging purposes only?  Should there be a `--verbose`?

Reply:
It's a debug print, I'll take it out.